[Snyk] Upgrade readable-stream from 3.6.1 to 3.6.2

inon-man opened the pull request:
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to upgrade readable-stream from 3.6.1 to 3.6.2.</h3>

:information_source: Keep your dependencies up-to-date. This makes it easier to fix existing vulnerabilities and to more quickly identify and fix newly disclosed vulnerabilities when they affect your project.
<hr/>

- The recommended version is **1 version** ahead of your current version.
- The recommended version was released **5 months ago**, on 2023-03-10.


<details>
<summary><b>Release notes</b></summary>
<br/>
  <details>
    <summary>Package name: <b>readable-stream</b></summary>
    <ul>
      <li>
        <b>3.6.2</b> - <a href="https://snyk.io/redirect/github/nodejs/readable-stream/releases/tag/v3.6.2">2023-03-10</a></br><h2>What's Changed</h2>
<ul>
<li>Fix es5 compatibility by <a class="user-mention notranslate" data-hovercard-type="user" data-hovercard-url="/users/mcollina/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="https://snyk.io/redirect/github/mcollina">@ mcollina</a> in <a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="1604489546" data-permission-text="Title is private" data-url="https://github.com/nodejs/readable-stream/issues/508" data-hovercard-type="pull_request" data-hovercard-url="/nodejs/readable-stream/pull/508/hovercard" href="https://snyk.io/redirect/github/nodejs/readable-stream/pull/508">#508</a></li>
</ul>
<p><strong>Full Changelog</strong>: <a class="commit-link" href="https://snyk.io/redirect/github/nodejs/readable-stream/compare/v3.6.1...v3.6.2"><tt>v3.6.1...v3.6.2</tt></a></p>
      </li>
      <li>
        <b>3.6.1</b> - <a href="https://snyk.io/redirect/github/nodejs/readable-stream/releases/tag/v3.6.1">2023-02-23</a></br><h2>What's Changed</h2>
<ul>
<li>fix undefined global in some environments (v3.x branch) by <a class="user-mention notranslate" data-hovercard-type="user" data-hovercard-url="/users/smeng9/hovercard" data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" href="https://snyk.io/redirect/github/smeng9">@ smeng9</a> in <a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="1516917876" data-permission-text="Title is private" data-url="https://github.com/nodejs/readable-stream/issues/502" data-hovercard-type="pull_request" data-hovercard-url="/nodejs/readable-stream/pull/502/hovercard" href="https://snyk.io/redirect/github/nodejs/readable-stream/pull/502">#502</a></li>
</ul>
<p><strong>Full Changelog</strong>: <a class="commit-link" href="https://snyk.io/redirect/github/nodejs/readable-stream/compare/v3.6.0...v3.6.1"><tt>v3.6.0...v3.6.1</tt></a></p>
      </li>
    </ul>
    from <a href="https://snyk.io/redirect/github/nodejs/readable-stream/releases">readable-stream GitHub release notes</a>
  </details>
</details>


<details>
  <summary><b>Commit messages</b></summary>
  </br>
  <details>
    <summary>Package name: <b>readable-stream</b></summary>
    <ul>
      <li><a href="https://snyk.io/redirect/github/nodejs/readable-stream/commit/6c32003bd8607da54f8ca1b096c4411778b060bc">6c32003</a> Bumped v3.6.2</li>
      <li><a href="https://snyk.io/redirect/github/nodejs/readable-stream/commit/c0b013c00de4f9a4742d8011a853daefce1a8072">c0b013c</a> Fix es5 compatibility (#508)</li>
    </ul>

   <a href="https://snyk.io/redirect/github/nodejs/readable-stream/compare/24b22f69f7b31eab1d24a3b8b136cf88f9cc7a38...6c32003bd8607da54f8ca1b096c4411778b060bc">Compare</a>
  </details>
</details>
<hr/>

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open upgrade PRs.*

For more information:  <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiIwMjU4YTU2NC1jYTg4LTQ3ZWUtOWQ1My0xNGVjMjBjMGYyMTMiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6IjAyNThhNTY0LWNhODgtNDdlZS05ZDUzLTE0ZWMyMGMwZjIxMyJ9fQ==" width="0" height="0"/>

🧐 [View latest project report](https://app.snyk.io/org/inon-man/project/9aba2789-1914-423c-abfa-5770e497b9ad?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🛠 [Adjust upgrade PR settings](https://app.snyk.io/org/inon-man/project/9aba2789-1914-423c-abfa-5770e497b9ad/settings/integration?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🔕 [Ignore this dependency or unsubscribe from future upgrade PRs](https://app.snyk.io/org/inon-man/project/9aba2789-1914-423c-abfa-5770e497b9ad/settings/integration?pkg&#x3D;readable-stream&amp;utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr#auto-dep-upgrades)

<!--- (snyk:metadata:{"prId":"0258a564-ca88-47ee-9d53-14ec20c0f213","prPublicId":"0258a564-ca88-47ee-9d53-14ec20c0f213","dependencies":[{"name":"readable-stream","from":"3.6.1","to":"3.6.2"}],"packageManager":"npm","type":"auto","projectUrl":"https://app.snyk.io/org/inon-man/project/9aba2789-1914-423c-abfa-5770e497b9ad?utm_source=github&utm_medium=referral&page=upgrade-pr","projectPublicId":"9aba2789-1914-423c-abfa-5770e497b9ad","env":"prod","prType":"upgrade","vulns":[],"issuesToFix":[],"upgrade":[],"upgradeInfo":{"versionsDiff":1,"publishedDate":"2023-03-10T09:02:34.449Z"},"templateVariants":[],"hasFixes":false,"isMajorUpgrade":false,"isBreakingChange":false,"priorityScoreList":[]}) --->

ZaradarBH commented:
@inon-man would like to keep the classic-terra changes, so we will close this PR and move the proposal stuff over to a new PR :)

snyk-bot committed:
fix: upgrade readable-stream from 3.6.1 to 3.6.2

Snyk has created this PR to upgrade readable-stream from 3.6.1 to 3.6.2.

See this package in npm:
https://www.npmjs.com/package/readable-stream

See this project in Snyk:
https://app.snyk.io/org/inon-man/project/9aba2789-1914-423c-abfa-5770e497b9ad?utm_source=github&utm_medium=referral&page=upgrade-pr
